File: front-end/hospital-transport-web/src/types/patient.ts
export interface Patient {
  id: string;
  fullName: string;
  rg: string;
  cpf: string;
  age: number;
  birthDate: string;
  susCardNumber: string;
  phoneNumber: string;
  motherName: string;
  createdAt: string;
}

export interface CreatePatientRequest {
  fullName: string;
  rg: string;
  cpf: string;
  age: number;
  birthDate: string;
  susCardNumber: string;
  phoneNumber: string;
  motherName: string;
}

export interface PatientSearchResult {
  id: string;
  fullName: string;
  susCardNumber: string;
  cpf: string;
  displayText: string;
}

File: back-end/HospitalTransport/HospitalTransport.Domain/Entities/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HospitalTransport.Domain.Entities
{
    public class Patient : BaseEntity
    {
        public string FullName { get; set; } = string.Empty;
        public string RG { get; set; } = string.Empty;
        public string CPF { get; set; } = string.Empty;
        public int Age { get; set; }
        public DateOnly BirthDate { get; set; } 
        public string SusCardNumber { get; set; } = string.Empty;
        public string PhoneNumber { get; set; } = string.Empty;
        public string MotherName { get; set; } = string.Empty;

        // Relacionamentos
        public ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
        public ICollection<Appointment> AppointmentsAsCompanion { get; set; } = new List<Appointment>();
    }
}